src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents the block number of the address in the address book.
+ */
+public class Block {
+	private int blockNumber;
+	
+	public Block(int blockNumber){
+		this.blockNumber = blockNumber;
```

Comment:
you still using tabs!? please change your eclispe settings to replace tabs by spaces, like right now :-D